src/caret_analyze/infra/lttng/records_provider_lttng.py

```diff
@@ -452,23 +479,26 @@ def publish_records(
         Parameters
         ----------
         publisher : PublisherStructValue
-            target publisher
+            Target publisher.
 
         Returns
         -------
         RecordsInterface
-            Columns
+            Columns: (in the case of tilde publisher)
+            - [topic_name]/rclcpp_publish_timestamp
+            - [topic_name]/rcl_publish_timestamp (Optional)
+            - [topic_name]/dds_write_timestamp (Optional)
+            - [topic_name]/message_timestamp
+            - [topic_name]/source_timestamp
+            - [topic_name]/tilde_publish_timestamp
+            - [topic_name]/tilde_message_id
 
+            Columns: (for cases other than tilde publisher)
```

Comment:
The format of documents created by mkdocs appears to be corrupted.
Please check and correct this.